CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contributing to Plex Media Player
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to Plex Media Player.
+
+#### 1 - Read the Plex CLA
+
+Before commiting, you will need to read the CLA which can be found in our [wiki](https://github.com/plexinc/plex-media-player/wiki/Contributor-License-Agreement#individual-agreement)
+
+Your commit message should include a sign off to confirm you've accepted the CLA by ending the commit with :
```

Comment:
`Your commit message should include a sign off to confirm you've accepted the CLA by ending the commit with:`